src/Scripting/VisualBasicTest/BasicScriptingTest.vbproj

```diff
@@ -8,8 +8,8 @@
     <PlatformTarget>AnyCPU</PlatformTarget>
     <OutputType>Library</OutputType>
     <AssemblyName>Microsoft.CodeAnalysis.VisualBasic.Scripting.UnitTests</AssemblyName>
-    <TargetFramework>netstandard1.3</TargetFramework>
-    <RoslynProjectType>UnitTestDesktop</RoslynProjectType>
+    <TargetFramework>net46</TargetFramework>
```

Comment:
So we'll no longer run VB scripting tests cross-platform? I'm fine with this, I just want to make sure it's intentional.